lib/main_common.pm

```diff
@@ -567,6 +567,10 @@ sub load_extra_tests {
         loadtest "console/zypper_moo";
         loadtest "console/gpg";
         loadtest "console/shells";
+        # MyODBC-unixODBC not available on < SP2 and sle 15
+        if (sle_version_at_least('12-SP2')  && !(sle_version_at_least('15'))) {
```

Comment:
probably ` !(sle_version_at_least('15')` is also covering this but I would leave `!is_staging` just for safety

Reply:
I was told to remove !is_staging because it will break on the next sles12 release.